pkgs/development/tools/misc/texinfo/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchurl
+, perl
+, xz
+, libintl
+, libiconv
+, bash
+, gnulib
+, gawk
+, ncurses
+, procps
+, interactive ? false
+}:
+
+let
+  make-texinfo = (import ./common.nix) {
```

Comment:
```suggestion
  make-texinfo = (import ./make-texinfo.nix) {
```

Reply:
The latest changes to this PR include your suggestion, thanks, @AndersonTorres !

Reply:
But why? `common.nix`  is much more common convention: https://github.com/NixOS/nixpkgs/pull/289690#issuecomment-2149549761